### PR TITLE
feat(learn): migrate mongodb-mongoose instructions to learn

### PR DESCRIPTION
**Pull request by ShaunSHamilton**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Related to: #39720 

The only adjustments to the tests made are superficial.

Linked to boilerplate: https://github.com/freeCodeCamp/boilerplate-mongomongoose/pull/31

### CODE REVIEW

File: curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/install-and-set-up-mongoose.md [hunk 1]
```diff
@@ -6,41 +6,60 @@ forumTopicId: 301540
...
-```
+In this challenge, you will import the required projects, and connect to your Atlas database.
 
 </section>
 
+## Instructions
+
 <section id='instructions'>
 
+Add mongodb and mongoose to the project’s `package.json`. Then, require mongoose in `myApp.js`. Store your MongoDB Atlas database URI in the private `.env` file as `MONGO_URI`. Surround the the URI with single or double quotes, and make sure no space exists between both the variable and the `=`, and the value and `=`. Connect to the database using the following syntax:
```
Comment (Sky020) [inline curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/install-and-set-up-mongoose.md:20]:
```suggestion
Add `mongodb` and `mongoose` to the project’s `package.json`. Then, require mongoose as `mongoose` in `myApp.js`. Store your MongoDB Atlas database URI in a private `.env` file as `MONGO_URI`. Surround the the URI with single or double quotes, and make sure no space exists between both the variable and the `=`, and the value and `=`. Connect to the database using the following syntax:
```

**Review by moT01 — DISMISSED** [review]
I am going to approve this since the PR does what it's supposed to do - moves any instructions from the boilerplate to here. But I really think they could use some help. The second lesson really got me lost. Perhaps something for another PR. Did you go through these at all @Sky020? Any thoughts on that.

**Comment by Sky020** [comment]
> The second lesson really got me lost. Perhaps something for another PR. Did you go through these at all @Sky020? Any thoughts on that.

@moT01 The _Create a Model_ lesson? To be honest, I cannot remember how much of this I went through, but nothing has changed...The camper is still required to go to the Mongoose docs to see how to structure a schema.

I suppose this line could be changed from:
> Now, create a model called `Person` from the `personSchema`.

To:
> Now, use the `Person` variable to create a model from the `personSchema` as per the documentation.

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@Sky020 Just a few suggestions and then in future PRs, we can clean up the verbiage of a few of the challenges a bit more.

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
@moT01 If you are ok with this PR, go ahead and merge it.  I already approved https://github.com/freeCodeCamp/boilerplate-mongomongoose/pull/31.

**Review by moT01 — APPROVED** [review]
Thanks for working on this @Sky020 🎉

**Pull request merged**